src/strategies/snet-farmers/index.ts

```diff
@@ -0,0 +1,56 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.1';
+
+type UserInfoResponse = {
+  amount: BigNumber;
+  rewardDebt: BigNumber;
+};
+
+type FarmResult = Record<string, UserInfoResponse>;
+
+const farmingAbi = [
+  'function userInfo(uint256 poolid, address account) external view returns (uint256 amount, int256 rewardDebt)'
+];
+
+const parseNumber = (value: BigNumberish): BigNumber => BigNumber.from(value);
+
+const multiCallerFactory = (network, provider, blockTag) => (abi) =>
+  new Multicaller(network, provider, abi, { blockTag });
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = 'latest';
```

Comment:
Same issue as https://github.com/snapshot-labs/snapshot-strategies/pull/202#discussion_r758960400

Reply:
fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/204/commits/f2f7c0ad9098a32bef0c3c48c7d456fc7852bc06